app/src/main/java/by/homework/hlazArseni/calculator/Fragment.kt

```diff
@@ -0,0 +1,88 @@
+package by.homework.hlazArseni.calculator
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import by.homework.hlazArseni.calculator.button_functions.allClearAction
+import by.homework.hlazArseni.calculator.button_functions.backSpaceAction
+import by.homework.hlazArseni.calculator.button_functions.equalsAction
+import by.homework.hlazArseni.calculator.button_functions.symbolAction
+import by.homework.hlazArseni.calculator.databinding.FragmentBinding
+
+
+class Fragment : Fragment() {
+
+    private var _binding: FragmentBinding? = null
+    private val binding get() = requireNotNull(_binding)
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return FragmentBinding.inflate(inflater, container, false)
+            .also { _binding = it }
+            .root
+
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        with(binding) {
+            val clickListener = View.OnClickListener {
+                when (it.id) {
+                    button0.id -> symbolAction(it, this)
+                    button1.id -> symbolAction(it, this)
+                    button2.id -> symbolAction(it, this)
+                    button3.id -> symbolAction(it, this)
+                    button4.id -> symbolAction(it, this)
+                    button5.id -> symbolAction(it, this)
+                    button6.id -> symbolAction(it, this)
+                    button7.id -> symbolAction(it, this)
+                    button8.id -> symbolAction(it, this)
+                    button9.id -> symbolAction(it, this)
```

Comment:
копипаст, можно через запятую указывать
```suggestion
                    button0.id, ->
                    button1.id -> symbolAction(it, this)
                    ...
```